client/state/ui/editor/media/imageEditor/actions.js

```diff
@@ -2,8 +2,11 @@
  * Internal dependencies
```

Comment:
The directory in which these files are contained should not be camel-cased, but rather dash-delimited, i.e. `image-editor`.